runtime/src/bank.rs

```diff
@@ -5310,8 +5328,23 @@ impl Bank {
         if new_feature_activations.contains(&feature_set::raise_block_limits_to_100m::id()) {
             let block_cost_limit = simd_0286_block_limits();
             let mut cost_tracker = self.write_cost_tracker().unwrap();
-            let account_cost_limit = cost_tracker.get_account_limit();
+            let account_cost_limit = if self
+                .feature_set
+                .is_active(&feature_set::raise_account_cu_limit::id())
+            {
+                (block_cost_limit as f64 * 0.4) as u64
```

Comment:
stated above - would prefer just a single fn in block_limits where we calculate this (however we choose to).

already asking you to change how the calc is done now 😉

Reply:
Yeah, good call. I'll pull this into a function